addons/mail/static/src/new/core_ui/message_notification_popover.xml

```diff
@@ -5,7 +5,7 @@
         <div class="o-mail-message-notification-popover m-2">
             <div t-foreach="props.message.notifications" t-as="notification" t-key="notification.id">
                 <i class="me-2" t-att-class="notification.statusIcon" t-att-title="notification.statusTitle" role="img"/>
-                <span t-if="notification.partner" t-esc="notification.partner.name"/>
+                <span t-if="notification.persona" t-esc="notification.persona.name || notification.persona.displayName"/>
```

Comment:
Isn't the displayName always undefined if the name is?
I wonder if the condition should be something like `notification.persona.displayName`.